src/Taskview/SearchBar.ts

```diff
@@ -920,17 +925,19 @@ export class SearchBar {
           .enter()
           .append('div')
           .attr('class', 'detail-info-option option-element')
-          .attr('data-optid', (d: IDataSubtypeConfig) => {
-            return this._composeGeneDataTypeOptId(optionId, d.id);
+          .attr('data-optid', (v: IDataSubtypeConfig) => {
+            return this._composeGeneDataTypeOptId(optionId, v.id);
           })
-          .classed('option-selected', (d: IDataSubtypeConfig) => {
-            return badgeIds.indexOf(this._composeGeneDataTypeOptId(optionId, d.id)) !== -1;
+          .classed('option-selected', (v: IDataSubtypeConfig) => {
+            return badgeIds.indexOf(this._composeGeneDataTypeOptId(optionId, v.id)) !== -1;
           })
-          .html((d: IDataSubtypeConfig) => d.name)
-          .on('click', (event, d: IDataSubtypeConfig) => {
-            const badgeName = this._composeGeneDataTypeName(data.optionText, d.name);
+          .html((v: IDataSubtypeConfig) => v.name)
+          .on('click', (event, v: IDataSubtypeConfig) => {
```

Comment:
v.name is not available, so access it from the d.name in the above context.
This needed an after fix as the eslint suggestion caused a bug.